mirage/serializers/package.js

```diff
@@ -0,0 +1,39 @@
+import { Serializer } from 'mirage-server';
+
+export default Serializer.extend({
+  embed: true,
+
+  include: ['customCoverage', 'vendor', 'visibilityData'],
+
+  serialize(response) {
+    let json = Serializer.prototype.serialize.apply(this, arguments);
```

Comment:
Is it a syntax error to use `this.super.serialize(response)`

Reply:
http://www.ember-cli-mirage.com/docs/v0.3.x/serializers/#serializeresponse-request Mirage borrows Backbone's implementation of extend, so `this.super.serialize(response)` ends up throwing `TypeError: Cannot read property 'serialize' of undefined`